straxen/plugins/event_area_per_channel.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import strax
+import straxen
+export, __all__ = strax.exporter()
+
+@export
+@strax.takes_config(
+    strax.Option('n_tpc_pmts', type=int,
+                 help='Number of TPC PMTs'),
+)
+class event_area_per_channel(strax.LoopPlugin):
+    """
+    Simple plugin that provides area per channel for main 
+    and alternative S1/S2 in the event. 
+    """
+    depends_on = ('events', 'event_basics', 'peaks')
```

Comment:
Unless I'm mistaken I don't think you need `events`

Reply:
Removed "events" from the list